apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -17,6 +25,167 @@ export default class ProjectCard extends React.Component {
     isDetailView: PropTypes.bool,
   };
 
+  constructor(props) {
```

Comment:
Out of curiosity, what do you think about refactoring this component to function component?
I would refactor this component to be function component (with hooks) and it would become cleaner for example.

Reply:
Definitely a good thought! I'm not sure if its worth doing right now, just with the limited time to get this out.